src/odemis/util/linalg.py

```diff
@@ -252,8 +252,8 @@ def generate_triangulation_points(max_dist: float, area_coords: Iterable[float])
     # away from the border of the given area
     length_x = abs(xmax - xmin)
     length_y = abs(ymax - ymin)
-    points_x = numpy.floor(length_x / max_dist) + 1
-    points_y = numpy.floor(length_y / max_dist) + 1
```

Comment:
Maybe add a small comment as to what issue rises when float values are used instead of integers

Reply:
in line 267, numpy.linspace expects the third input argument to be int. should I add a comment to explain it?

Reply:
Ah no I think that is pretty straightforward 'as is'. You can ignore my comment.